src/test/java/teammates/test/cases/browsertests/FeedbackContributionQuestionUiTest.java

```diff
@@ -180,7 +180,7 @@ private void testCopyOptions() {
         feedbackEditPage.clickNewQuestionButton();
         feedbackEditPage.selectNewQuestionType("CONTRIB");
         feedbackEditPage.fillQuestionTextBoxForNewQuestion("q2, contribution qn");
-        assertEquals("Please select a visibility option",
```

Comment:
The behaviour should follow that of other question types - when adding it as the first question in an empty session, it should show `Please select a visibility option` to emphasise the fact that the instructor should choose an option. Previous discussion [here](https://github.com/TEAMMATES/teammates/pull/6669#discussion_r105699337).